templates/pages/core/documentation.md

```diff
@@ -5,4 +5,9 @@ description: Everything there is to know about building and managing Ubuntu Core
 
 # Documentation
 
-Documentation introduction
+The documentation is the reference material for all things Ubuntu Core . Use the documentation when you want to understand the key concepts behind Ubuntu Core, the development process to go from an idea to a device in production or simply to get the full API or command line reference.
```

Comment:
please:

 1. remove extra space after 'Ubuntu Core . Use' before the full stop
 2. add a comma before the 'or' in 'production or simply'